doc/cprover-manual/loop-invariant-synthesizer.md

```diff
@@ -0,0 +1,37 @@
+[CPROVER Manual TOC](../)
+
+## Loop Invariant Synthesis with goto-instrument
+
+This is a loop invariant synthesizer built in goto-instrument that 
+synthesize [loop invariants](contracts-invariants.md) for a 
+pre-existing GOTO binary, annotate the invariant clause into 
+the GOTO binary, and produce an annotated binary as output.
```

Comment:
Is it really most important to talk about `goto-instrument` here? I'd expect that users first want to know a sentence about what loop invariants are (and, specifically, what kinds of loop invariants are being synthesized - inductive ones?), and then what capabilities/limitations this synthesizer has (it's an undecidable problem).